features/background-origin.yml

```diff
@@ -0,0 +1,10 @@
+name: background-origin
+description: The `background-origin` CSS property sets the background origin position relative to the border and padding of an element.
```

Comment:
Suggestion to avoid circularity:

```suggestion
description: The `background-origin` CSS property sets the background starting position relative to the border and padding of an element.
```